distribution/docs/src/main/resources/_contents/_catalog-contents/integrating-catalog-contents.adoc

```diff
@@ -786,285 +780,13 @@ None.
 
 ==== OpenSearch Endpoint
 
-The OpenSearch Endpoint provides a CDR REST Search v3.0 and CDR REST Brokered Search 1.1 compliant ${branding} endpoint that a client accesses to send query parameters and receive search results.
-
-This endpoint uses the input query parameters to create an OpenSearch query.
-The client does not need to specify all of the query parameters, only the query parameters of interest.
-
-This endpoint is a `JAX-RS` RESTful service and is compliant with the CDR IPT BrokeredSearch, CDR IPT OpenSearch, and OpenSearch specifications.
-
-===== Installing and Uninstalling
-
-The OpenSearch Endpoint can be installed and uninstalled using the normal processes described in the  Configuring ${branding} section.
-
-===== Configuring
-
-The OpenSearch Endpoint has no configurable properties.
-It can only be installed or uninstalled.
 
 ===== Using the OpenSearch Endpoint
 
-Once installed, the OpenSearch endpoint is accessible from `http://<${ddf-branding}_HOST>:<${ddf-branding}_PORT>/services/catalog/query`.
+Once installed,
```

Comment:
Just saw the file name. I should add to the description that this file is being picked apart. The `Using the OpenSearch Endpoint` info is now in `_endpoints/opensearch-endpoint-contents.adoc`.